wicker/core/config.py

```diff
@@ -40,6 +40,7 @@ def from_json(cls, data: Dict[str, Any]) -> BotoS3Config:
 
 @dataclasses.dataclass(frozen=True)
 class WickerAwsS3Config:
+    loaded: bool = False
```

Comment:
I know it is not introduced by you, but I think the WickerAwsS3Config should be singleton instead of class.